client/src/components/MultiJurisdictionAudit/Progress/ProgressMap.tsx

```diff
@@ -0,0 +1,353 @@
+        return (
+          jurisdictionNameLower === countyNameLower ||
+          jurisdictionNameLower.replace(' county', '') === countyNameLower
+        )
+      })
+
+      if (filteredJurisdiction) {
+        const jurisdictionStatus = getJurisdictionStatus(filteredJurisdiction)
+        switch (jurisdictionStatus) {
+          case JurisdictionProgressStatus.UPLOADS_COMPLETE:
+          case JurisdictionProgressStatus.AUDIT_COMPLETE:
+            return 'success'
+          case JurisdictionProgressStatus.UPLOADS_FAILED:
+            return 'danger'
+          case JurisdictionProgressStatus.AUDIT_IN_PROGRESS:
+            return 'progress'
+          case JurisdictionProgressStatus.UPLOADS_NOT_STARTED:
+          case JurisdictionProgressStatus.AUDIT_NOT_STARTED:
+            return 'gray'
+          default:
+            return 'default'
+        }
+      }
+      // incase of non-participant jurisdictions
+      return 'default'
+    },
+    [jurisdictions]
+  )
+
+  useEffect(() => {
+    const loadMapData = async () => {
+      let respJsonData: Topology | undefined
+      try {
+        respJsonData = await json(
+          'https://cdn.jsdelivr.net/npm/us-atlas@3/counties-10m.json'
+        )
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(`File Error: ${error}`)
+        // use local topojson file if CDN link fails to load
+        respJsonData = await json('/us-states-counties.json')
+      }
+      setJsonData(respJsonData)
+    }
+    loadMapData()
+  }, [])
+
+  useEffect(() => {
+    const projection = geoAlbers()
+    const path = geoPath().projection(projection)
+
+    if (jsonData) {
+      const svgElement = select(d3Container.current)
+
+      svgElement.selectAll('path').remove()
+
+      const usState = (feature(
+        jsonData,
+        jsonData.objects.states
+      ) as GeoJSON.FeatureCollection).features.filter(
+        d =>
+          d &&
+          d.properties &&
+          stateName &&
+          d.properties.name === getStateName(stateName)
+      )[0]
+
+      // county ID's initial 2 characters are of state
+      // hence, just setting counties of audit state
+      const usCounties = (feature(
+        jsonData,
+        jsonData.objects.counties
+      ) as GeoJSON.FeatureCollection).features.filter(
+        d =>
+          d &&
+          d.id &&
+          usState &&
+          usState.id &&
+          d.id.toString().slice(0, 2) === usState.id.toString()
+      )
+
+      const jurisdictionNames = jurisdictions.map(jurisdiction =>
+        jurisdiction.name
+          .toLowerCase()
+          .replace('county', '')
+          .trim()
+      )
+      const filteredCounties = Object.values(usCounties).filter(county =>
+        jurisdictionNames.includes(
+          county.properties && county.properties.name.toLowerCase()
+        )
+      )
+
+      if (filteredCounties.length / jurisdictions.length >= 0.5) {
```

Comment:
is there a reason for hiding the map within this `useEffect`? seems like it would be a bit simpler to just not render the map at all if we don't meet the threshold (similar to what you did with the labels below)